{
  "schema_version": "1.4.0",
  "id": "GHSA-4qp3-mprp-ccrj",
  "modified": "2022-06-16T00:00:27Z",
  "published": "2022-06-09T00:00:16Z",
  "aliases": [
    "CVE-2022-30882"
  ],
  "details": "pyanxdns package in PyPI version 0.2 is vulnerable to code execution backdoor. The impact is: execute arbitrary code (remote). When installing the pyanxdns package of version 0.2, the request package will be installed.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-30882"
    },
    {
      "type": "WEB",
      "url": "https://github.com/egeback/pyanxdns/issues/1"
    },
    {
      "type": "WEB",
      "url": "https://pypi.org/project/pyanxdns"
    },
    {
      "type": "WEB",
      "url": "http://pypi.doubanio.com/simple/request"
    }
  ],
  "database_specific": {
    "cwe_ids": [],
    "severity": "CRITICAL",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2022-06-08T20:15:00Z"
  }
}